appengine/bigquery/README.md

```diff
@@ -2,18 +2,24 @@
 
 This sample demonstrates [authenticating to BigQuery in App Engine using OAuth2](https://cloud.google.com/bigquery/authorization).
 
-### Setup
+### Running the sample
 
-* To install dependencies for this sample, run:
+1. To install dependencies for this sample, run:
 
-    $ pip install -t lib -r requirements.txt
+        $ pip install -t lib -r requirements.txt
 
-* You must then update `main.py` and replace `<myproject_id>` with your project's
-  id.
-* You'll need a client id from your project - instructions
-  [here](https://cloud-dot-devsite.googleplex.com/bigquery/authorization#clientsecrets).
-  Once you've downloaded the client's json secret, copy it to the root directory
-  of this project, and rename it to `client_secrets.json`.
-* You can then run the sample on your development server:
+2. You must then update `main.py` and replace `<your-project-id>` with your project's
+   ID.
 
-    $ dev_appserver.py .
+3. You'll need a client id from your project - instructions
+   [here](https://cloud.google.com/bigquery/authorization#clientsecrets).
```

Comment:
Since you're here, can you change '/authorization' to '/authentication'? (...it's a long story..)